### ISSUE TITLE
Error while calling colony integrator getRequests()

### ISSUE THREAD

**Issue by Hiranus** [Seniorendi/AdvancedPeripherals#302]
### Descripe

I wrote code that integrates RS with minecolonies request, and it worked quite well
https://pastebin.com/Db8kbsyA
Lately this code crashes when line 72 is executed, which is` colonyIntegrator.getRequests()` line.

### Steps to reproduce

![obraz](https://user-images.githubusercontent.com/6943031/169788567-78b6eb78-8399-4eb0-8770-37eb1da1777b.png)

1. Wrap Colony Integrator
2. Try to execute getRequests()

### Multiplayer?

Yes

### Version

1.16.5-0.7.8r (Latest 1.16)

### Minecraft, Forge and maybe other related mods versions

forge 36.2.34 MC 1.16.5

### Screenshots or Videos

https://imgur.com/a/gvx0jKS

### Crashlog/log
Full log from server is too big for pastebin, and this is only interesting part.
https://pastebin.com/ZU0r0X75

**Comment by DasBrain** [comment]
The issue lies IMHO here:

https://github.com/Seniorendi/AdvancedPeripherals/blob/8bb1196585d874acb4573a3cf187cb8937b2ab22/src/main/java/de/srendi/advancedperipherals/common/addons/computercraft/peripheral/ColonyPeripheral.java#L246

Change this line to:

```java
if (request != null && request.getRequest() instanceof IDeliverable)
```

**Comment by SirEndii** [comment]
I'll test that, compile it and send the compiled file in here
But thank you for the hint (:

**Comment by DasBrain** [comment]
Yeah, I didn't get to test it - setting up a new project to compile is a hassle.

Anyway, to reproduce:
* Build a Stash (from minecolonies) in a colony without a working Journyman.
* Put some items into the Stash.
* A new request is created to move the items from the Stash into the warehouse, but no Journyman will pick it up.

(My script breaks once the Journyman gets killed, falls ill...)

Reason behind this suggestion:
Later `request.getRequest()` is casted to `IDeliverable` - which fails:
https://github.com/Seniorendi/AdvancedPeripherals/blob/8bb1196585d874acb4573a3cf187cb8937b2ab22/src/main/java/de/srendi/advancedperipherals/common/addons/computercraft/peripheral/ColonyPeripheral.java#L252

**Comment by SirEndii** [comment]
I guess this changed after I made this
Worked as intended, but minecolonies does a lot of changes

**Comment by SirEndii** [comment]
Hey
I fixed the bug and added compatibility for minecolonies 1.0.799-ALPHA.
[advancedperipherals-1.16.5-0.7.8r.zip](https://github.com/Seniorendi/AdvancedPeripherals/files/8803810/advancedperipherals-1.16.5-0.7.8r.zip)
Here is the file is someone want to test it.
Unzip the file before using.

**Comment by SirEndii** [comment]
Already fixed in 1.16.5-0.7.9r
